github/repos.go

```diff
@@ -658,10 +630,10 @@ type DismissalRestrictions struct {
 // separate from DismissalRestrictions above because the request structure is
 // different from the response structure.
 type DismissalRestrictionsRequest struct {
-	// The list of user logins who can dismiss pull request reviews. (Required; use []string{} instead of nil for empty list.)
-	Users []string `json:"users"`
-	// The list of team slugs which can dismiss pull request reviews. (Required; use []string{} instead of nil for empty list.)
-	Teams []string `json:"teams"`
+	// The list of user logins who can dismiss pull request reviews. (Required; use nil to disable dismissal_restrictions or []string{} otherwise.)
```

Comment:
`&[]string{}` here, not `[]string{}`.

Also, it might be worth adding a note saying that either both `Users` and `Teams` must be nil, or both must be non-nil.